docs/developer/url-path-structure.md

```diff
@@ -4,18 +4,55 @@ We've implemented the following structure:
 
 | URL path | Description  |
 |-|-|
-| `/categories` | List view |
-| `/categories?mode=bulkEdit` | List view in bulk edit mode* |
-| `/categories/:id` | Edit item view |
-| `/categories/add` | Create new item form |
+| `/` | Redirects to `/overview` |
+| `/overview` | Shows tiles for all available sections |
+| `/overview/data-elements` | Section view with tiles |
+| `/data-elements` | Data elements list view |
+| `/data-elements/:id/details` | Data element details view |
+| `/data-elements/:id/edit` | Data element edit view |
```

Comment:
Since we don't have a dedicated "Details view" we can just let `/:id` be the edit view
```suggestion
| `/data-elements/:id` | Data element edit view |
```